### DISCUSSION TITLE
Predefined configuration for compose up

### DISCUSSION THREAD

**Discussion by MarcinGil** [featurehub-io/featurehub#1146] [General]
Hi!
I'm trying out Feature Hub, however I would like to have a predefined set of data (users, environments, etc) - that can be ingested at first run of the docker image.

Is it possible to predefine this in the file or is it necessary to go through first run/setup and I need to commit the .h2 database into the repository?

Basically I would like to write a simple app that uses all the features of FH and be able to just give away the repo url for everyone to play/check out.

**Comment by IrinaSouth** [comment]
Hi @MarcinGil,
This is a good idea, thanks for suggesting this.
The way I can think of is to commit .h2 database (it is just a file) as you mentioned. And then for users to clone the repo. Another way we use to populate the database is automated test/script. This can be done via using FH Admin SDK API. We have some examples of automated tests here https://github.com/featurehub-io/featurehub/blob/main/e2e/test/features/applications.feature  It is probably more work for you however, so maybe .h2 already populated database is easier.
